nomad/settings.py

```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = (os.environ.get('RUNNING_PROD') is None)
 
-ALLOWED_HOSTS = ['localhost', '.herokuapp.com']
+ALLOWED_HOSTS = ['localhost', '.herokuapp.com','127.0.0.1']
```

Comment:
Doesn't `localhost` serve same purpose as `127.0.0.1 `

Reply:
yeah...actually I missed to remove that change.